config/covid-19/test/check-app-settings-is-committed

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
```

Comment:
@newtewt is this one making sense for any new configuration in CHT-Core? I saw it in default and standard and decided to copy the code here as well.

Reply:
Yeah, it should probably be moved out of the configs and into the `cht-core/scripts/` directory. Also updated to `cht` instead of `medic-conf`

Reply:
I see that default config is slightly different by not having the `--local` [flag](https://github.com/medic/cht-core/blob/master/config/default/test/check-app-settings-is-committed)
I'd like to keep this PR targeted with only FIND Grant changes for the record, so moving the file to `cht-core/script` can be done in another PR. 
But I'm going to update this one to use `cht` instead of `medic-conf` :)

Reply:
Done